src/Database/DatabaseManager.cs

```diff
@@ -167,8 +169,7 @@ internal static Assembly[] LoadAssemblyFromPlugInsDirectory(IFileSystem fileSyst
 
             foreach (var plugin in plugins)
             {
-                var asesmblyeData = fileSystem.File.ReadAllBytes(plugin);
-                assemblies.Add(Assembly.Load(asesmblyeData));
+                assemblies.Add(Assembly.LoadFrom(plugin));
```

Comment:
Might be worth silencing this warning here and adding an explanation on the code so it doesn't accidently get changed in the future.

Reply:
done now